### PR TITLE
replace ‘changelog’ microcopy with meaningful text

### PR DESCRIPTION
**Pull request by gazaston**

### CODE REVIEW

**Comment by gazaston** [comment]
[Changelog microcopy](https://trello.com/c/d35uMGM3/72-changelog-microcopy)

File: app/views/developments/_form.html.haml [hunk 1]
```diff
@@ -12,4 +12,4 @@
 
 - if @development.audit_changes?
   .govuk-form-group
-    = f.input :audit_comment, label: 'Changelog'
+    = f.input :audit_comment, label: 'Reason for changes to legal agreement'
```
Comment (james) [inline app/views/developments/_form.html.haml]: Should we add `, required: true` to this too?

Comment (gazaston) [reply app/views/developments/_form.html.haml]: I wondered that — perhaps it belongs to https://trello.com/c/g1oNFIu3/59-editing-a-development-required-form-hint ?

Comment (james) [reply app/views/developments/_form.html.haml]: ah yes

**Pull request merged**